addons/website_sale/views/snippets/snippets.xml

```diff
@@ -80,7 +80,7 @@
                          data-customize-website-views="website_sale.products_description"
                          data-no-preview="true"
                          data-reload="/"/>
-            <we-row string="Left Panel" class="o_we_full_row">
+            <we-row id="left-panel" string="Left Panel" class="o_we_full_row">
```

Comment:
Should be more specific since ids are supposed to be unique, this is too generic, not sure whether `-` or `_` is better fit 
```suggestion
            <we-row id="o_wsale_grid_left_panel" string="Left Panel" class="o_we_full_row">
```